packages/apps/nats/templates/nats.yaml

```
@@ -18,6 +18,13 @@ spec:
     nats:
       fullnameOverride: {{ .Release.Name }}
       config:
+        {{- if and .Values.username .Values.password }}
+        merge:
```

yamllint

[warning] wrong indentation: expected 6 but found 8

(indentation)